build.sbt

```diff
@@ -342,16 +346,36 @@ lazy val scioCassandra3: Project = Project(
   scioTest % "it"
 ).configs(IntegrationTest)
 
-lazy val scioElasticsearch: Project = Project(
-  "scio-elasticsearch",
-  file("scio-elasticsearch")
+lazy val scioElasticsearch2: Project = Project(
+  "scio-elasticsearch2",
+  file("scio-elasticsearch2")
 ).settings(
   commonSettings,
   description := "Scio add-on for writing to Elasticsearch",
   libraryDependencies ++= Seq(
     "com.google.guava" % "guava" % guavaVersion,
     "joda-time" % "joda-time" % jodaTimeVersion,
-    "org.elasticsearch" % "elasticsearch" % elasticsearchVersion
+    "org.elasticsearch" % "elasticsearch" % elasticsearch2Version
+  )
+).dependsOn(
+  scioCore,
+  scioTest % "test"
+)
+
+lazy val scioElasticsearch5: Project = Project(
+  "scio-elasticsearch5",
+  file("scio-elasticsearch5")
+).settings(
+  commonSettings,
+  description := "Scio add-on for writing to Elasticsearch",
+  libraryDependencies ++= Seq(
```

Comment:
Are both `log4j-api` & `log4j-core` needed then? If it behaves like `slf4j` then without core it'll just fall back to no-op? Users can supply their own impl then.